server/database/seeds/users.js

```diff
@@ -0,0 +1,13 @@
+
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('table_name').del()
+    .then(function () {
+      return Promise.all([
+        // Inserts seed entries
+        knex('table_name').insert({id: 1, colName: 'rowValue1'}),
```

Comment:
going to throw errors since there isn't a table `table_name` and no column `colName`